### ISSUE TITLE
License

### ISSUE THREAD

**Issue by Aspie96** [atisawd/boxicons#111]
There are multiple Creative Commons licenses and multiple CC licenses in the 4.0 version.

Here they are: https://creativecommons.org/licenses/.

Most of them are non-free and they are widely different. "CC 4.0" licensed means absolutely nothing.

Still, the purpose of this repository is to have them under the Creative Commons Attribution 4.0 International Public License or, for short, CC BY-4.0 (notice the "BY"). I know this because of the package.json file specifying "CC-BY-4.0": this is literally the only clue I can see to which license we are talking about.

I suggest updating the readme to reflect this. Otherwise, it's very confusing.

Before fixing this, I suggest notifying all contributors to make sure they agree: this should be fairly easy as there are only 6 of them.

Thanks!

**Comment by arthuro555** [comment]
This is especially confusing since on the website there is a contradictory statement: https://boxicons.com/usage/#license  
As you can see, CC 4.0 links to CC 4.0 BY, but it says just below that attribution isn't required, which is in contradiction with the CC 4.0 BY license, where the BY means that attribution is required.

**Comment by yurkobb** [comment]
Yeah, and also it is unclear what kind of attribution is required for what kind of use. Is it enough to add a comment in the code where the icon is used saying something like `Icon by boxicons.com under cc by 4.0`? Or should one attribute in the footer of the website? What if there are different icon sets used, how attribute only specific icons? Etc...

**Comment by Aspie96** [comment]
Note: they cannot add requirements on the specific way of providing attribution because CC BY 4.0 does not allow adding restrictions (only exceptions: additional permissions).
Any attribution in compliacne with CC BY 4.0 will be fine.
